app/models/articles/feeds.rb

```diff
@@ -84,19 +123,22 @@ def self.lever_catalog
 
       relevancy_lever(:comments_count,
                       label: "Weight to give to the number of comments on the article.",
+                      range: "[0..∞)",
```

Comment:
😲

Reply:
It's probably not infinity but it's definitely more than a dozen